### ISSUE TITLE
Mobs killed by pet don't give drops

### ISSUE THREAD

**Issue by strangefolk1** [HellscreamWoW/Tracker#3462]
Very annoying when your pet is stronger and literally steals monsters from you.
This happens both for hunters and warlocks - only when the pet itself engages the target or delivers a certain % of the damage.

**Comment by phikaro** [comment]
I think that's normal, but I'm not sure though. =)

**Comment by silverjade** [comment]
That's right. On retail you need to do a third of the dmg on normal mobs to get loot from it should a pet kill it, on bosses, you need to hit it at least once before the pet kills it in order to loot it. If the pet kills it without you doing any dmg, you don't get any loot.

**Comment by strangefolk1** [comment]
I'm hitting the mobs with spells, but when it comes to multiple pulls my pet is doing most of the work. He's not soloing them though, I noticed this happens even if I put a debuff on the mobs or hit them for low damage. 
Meaning if I deal 20% of the damage and my pet 80%, approximately, I don't get the loot.
I strongly doubt it works this way on retail, but I haven't really checked.

**Comment by Collector-Hellscream** [comment]
http://eu.battle.net/wow/en/forum/topic/5847766569
I think it's same as retail.
